x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/translations.ts

```diff
@@ -143,6 +177,13 @@ export const DUPLICATE_RULE = i18n.translate(
   }
 );
 
+export const DUPLICATE_RULE_ERROR = i18n.translate(
+  'xpack.siem.detectionEngine.rules.allRules.actions.duplicateRuleErrorDescription',
+  {
+    defaultMessage: 'Error duplicating rule…',
```

Comment:
You want to use ellipses or three dots?

If so just to let you know we have these other two that aren't?

<img width="800" alt="Screen Shot 2020-01-13 at 1 52 03 PM" src="https://user-images.githubusercontent.com/1151048/72291140-fae3e180-360b-11ea-86d0-86df6c999837.png">

Reply:
Design has been having them as ellipses iirc, so I've been following that convention. @MichaelMarcialis, can you confirm which we should be conforming too?

Reply:
Great question. I'd prefer our usage be of a horizontal ellipsis (`…` or `&hellip;`) going forward.

Reply:
Perfect -- thanks @MichaelMarcialis! I've created https://github.com/elastic/kibana/issues/54677 for tracking so it doesn't get lost in the shuffle! 🙂